compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangParserListener.java

```diff
@@ -3401,6 +3401,16 @@ public void exitDeprecateAnnotationDescriptionLine(BallerinaParser.DeprecateAnno
         this.pkgBuilder.endDeprecateAnnotationDocumentationDescription(getWS(ctx), description);
     }
 
+    @Override
+    public void exitDeprecatedParametersDocumentation(BallerinaParser.DeprecatedParametersDocumentationContext ctx) {
+        if (isInErrorState) {
+            return;
+        }
+        String str = ctx.DeprecatedParametersDocumentation() != null ?
```

Comment:
Can this be null here?

Reply:
Yes, because the markdown documentation can be available without deprecated parameter documentation.

Reply:
But does `exitDeprecatedParametersDocumentation()` get invoked then?

Reply:
It does not get invoked when the deprecated params doc is not available. So null check is removed